riscv/tests/instruction_tests/README.md

```diff
@@ -9,65 +9,61 @@ kind of arithmetic over `.text` label and addresses, nor alignment or spacing
 directives in `.text` sections. Most unsupported instructions are related to
 this limitation.
 
+One consequence of this limitation is that all the supported uses of `auipc` and
+`lui`, when referring to a `.text` label, is when they can be fused with the
+next instruction to make use of the loaded address immediately (e.g. `addi` or
```

Comment:
I'll try to write it better. But using `auipc` and `lui` to load a `.text` address is only supported if it can be fused with the next instruction. It works fine if they are used to load a data address.